frontend/src/store/facilityBooking/action.ts

```diff
@@ -160,6 +199,16 @@ export const editMyBooking = (oldBooking: Booking) => (dispatch: Dispatch<Action
   })
 }
 
+/**
+ *
+ * Takes in a new tab and changes to that tab.
+ *
```

Comment:
yeah same, is straightforward so no need describe
```suggestion
 *
```